crates/nu_plugin_shuffle/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use nu_errors::ShellError;
```

Comment:
Rather than building a lib here, you want to build it as an executable binary.  That should fix the errors you're seeing on the PR.